mw-webapp/src/component/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,38 @@
+import {PropsWithChildren, ReactNode} from "react";
+import clsx from "clsx";
+import {PositionTooltip} from "src/component/tooltip/PositionTooltip";
+import styles from "src/component/tooltip/Tooltip.module.scss";
+
+/**
+ * Tooltip props
+ */
+interface TooltipProps {
+  /**
+  * Tooltip's content
+  */
+  content: string | ReactNode;
+  /**
+  * Tooltip's className
+  */
+  className?: string;
+  /**
+  * Tooltip's position
+  * default: {@link PositionTooltip.TOP}
+  */
+  position?: PositionTooltip;
+}
+
+export const Tooltip: React.FC<PropsWithChildren<TooltipProps>> = (props: PropsWithChildren<TooltipProps>) => {
```

Comment:
Where is ts-doc for public member?

Reply:
I don’t understand, when entering an example in the code, how it should look like, if I open any component in the project, it won’t be there, should I @public tag the function? Or are you talking about a property in the interface?